src/enum.ts

```diff
@@ -1,37 +1,79 @@
+/* Copied from https://github.com/mchehab/zbar/blob/5b3c33d5c87f58633b550d592f6eb8e4fd580b1e/include/zbar.h */
+
 export enum ZBarSymbolType {
-  ZBAR_NONE = 0 /**< no symbol decoded */,
-  ZBAR_PARTIAL = 1 /**< intermediate status */,
-  ZBAR_EAN8 = 8 /**< EAN-8 */,
-  ZBAR_UPCE = 9 /**< UPC-E */,
-  ZBAR_ISBN10 = 10 /**< ISBN-10 (from EAN-13). @since 0.4 */,
-  ZBAR_UPCA = 12 /**< UPC-A */,
-  ZBAR_EAN13 = 13 /**< EAN-13 */,
-  ZBAR_ISBN13 = 14 /**< ISBN-13 (from EAN-13). @since 0.4 */,
-  ZBAR_I25 = 25 /**< Interleaved 2 of 5. @since 0.4 */,
-  ZBAR_CODE39 = 39 /**< Code 39. @since 0.4 */,
-  ZBAR_PDF417 = 57 /**< PDF417. @since 0.6 */,
-  ZBAR_QRCODE = 64 /**< QR Code. @since 0.10 */,
-  ZBAR_CODE128 = 128 /**< Code 128 */,
-  ZBAR_SYMBOL = 0x00ff /**< mask for base symbol type */,
-  ZBAR_ADDON2 = 0x0200 /**< 2-digit add-on flag */,
-  ZBAR_ADDON5 = 0x0500 /**< 5-digit add-on flag */,
-  ZBAR_ADDON = 0x0700 /**< add-on flag mask */
+  ZBAR_NONE	     = 0,   /**< no symbol decoded */
```

Comment:
Could you help to do the `npm run format` to format the code.
We should either make them align or use only one space.

Reply:
I think this didn't work because you might run it on another commit.
Or you can run `git diff origin/master --name-only --diff-filter=ACMR "*.ts" "*.js" | sed 's| |\\ |g' | xargs npx prettier --write`.

Reply:
Ooops, the file contained a mixture of apces and tabs which I did not notice in VS Code. Fixed.

Reply:
... spaces ...